website/blog/2021-06-13-Hydra_1.1.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Hydra 1.1
+author: Omry Yadan
+author_title: Creator of Hydra
+author_url: https://github.com/omry
+author_image_url: https://graph.facebook.com/733244046/picture/?height=200&width=200
```

Comment:
I can't see your face -- not sure if intended :)

Reply:
Yeah, this no longer works.
Not sure if there is an easy way to get the Facebook profile picture.

Reply:
No idea => maybe you can put it on your personal Dropbox or Google drive? (unsure if that'd work though).
Feel free to resolve otherwise.

Reply:
I posted a question in the internal graph API group, let's see if they have a solution.
If they do we can fix this retroactively.